docs/guides/base-images.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: How to use parent images
```

Comment:
I think it might be good to add a couple sentences wrt. _why_ you might add a custom base image